ath79: add support for Ubiquiti UniFi AC-Pro

lucize opened the pull request:
rework the dts to a common unifi-ac dtsi
pro network is connected via phy0 and has usb ports
lite network is connected via phy4 without usb ports

everything works

Signed-off-by: Lucian Cristian <lucian.cristian@gmail.com>

juppin commented:
You could add `ATH_SOC := qca9563` and `DEVICE_PACKAGES += kmod-ath10k ath10k-firmware-qca988x` to `Device/ubnt-unifiac` and remove this from the device image targets.
And probably you should explain more what are you doing in with the commit message...

lucize commented:
yes, I should add more details to commit, I'll rework the image creation

lucize commented:
@juppin should be better now

juppin commented on target/linux/ath79/image/generic-ubnt.mk:
```diff
@@ -81,20 +81,25 @@ endef
 define Device/ubnt-unifiac-lite
   $(Device/ubnt-unifiac)
   DEVICE_TITLE := Ubiquiti UniFi AC-Lite
   DEVICE_PACKAGES := kmod-ath10k ath10k-firmware-qca988x
```
Why not remove this and do a `DEVICE_PACKAGES += kmod-usb-core kmod-usb2` on Device/ubnt-unifiac-pro

lucize replied:
yeah, I guess is better that way

lucize committed:
ath79: add support for Ubiquiti UniFi AC-Pro

rework the dts to a common unifi-ac dtsi
pro network is connected via phy0 and has usb ports
lite network is connected via phy4 without usb ports

Signed-off-by: Lucian Cristian <lucian.cristian@gmail.com>

mkresin commented:
Thanks! Merged to [my staging tree](https://git.openwrt.org/?p=openwrt/staging/mkresin.git;a=summary).

I've applied some more cosmetic changes. Would you please test an image build from my staging tree! I would like to ensure that I haven't broke anything.

lucize commented:
@mkresin I've built a clean image from your staging and all is working
